.gitattributes

```diff
@@ -0,0 +1,13 @@
+* text=auto
```

Comment:
I think this should be safe but haven't used it before. Opinions @doctrine/doctrinecore?

Reply:
yeah seems ok for this repo. it's all source code